chia/wallet/did_wallet/did_wallet.py

```diff
@@ -655,7 +668,10 @@ async def transfer_did(self, puzzle_hash: bytes32, fee: uint64) -> TransactionRe
 
     # The message spend can tests\wallet\rpc\test_wallet_rpc.py send messages and also change your innerpuz
     async def create_message_spend(
-        self, message_puz: Program, message_sol: Program = Program.to([]), new_innerpuzhash: Optional[bytes32] = None
+        self,
+        coin_announcements: Optional[Set[bytes]] = None,
```

Comment:
Are we only have these 2 types of condition?

Reply:
We can add more later if we need to